projects.html

```diff
@@ -0,0 +1,231 @@
+        </ul>
+    </nav>  
+       
+    
+
+
+
+<main class=" ">
+<nav class="mobile_nav">
+    <ul class="mobile_nav_links">
+        <li><a class="links"  href="index.html">ABOUT ME</a></li>
+        <li><a class="links"  href="resume.html">RESUME</a></li>
+        <li><a  class="links" href="projects.html">PROJECTS</a></li>
+        <li><a class="links"  href="contact.html">CONTACT</a></li>
+    </ul>
+</nav>
+    <section class="container-fluid  text-center">
+    <h2 class="lets-talk my-5  "> <img src="./img/logo.png" alt="logo"> Projects</h2>
+        
+    <div class=" container "> 
+         <div class="row">
+       </div> <p class="  mx-auto p-3 col-md-12  col-lg-6 col-sm-12  col-12 project-page-text">I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to
+        add your own content and make changes to the font. I’m a great place for you to tell a story and let your users know a
+        little more about you.</p>
+        </div>
+    </div>
+
+
+       <div class="container my-5">
+           <div class="projects-box mx-lg-auto mx-sm-auto mx-auto mx-md-auto">
+        <div class="row mx-lg-auto my-4 ">
+               <div class="col-lg-7 col-md-7  col-sm-12 col-12 bg-white card-right-color">
+                
+                
+                 <div class="row d-flex flex-lg-row  flex-md-row align-items-center  text-start ">
+                     <div class="col-lg-1  p-0 pt-5 mt-5 blue-box"></div>                   
+                   
+                     <div class=" small-project-font large-project-font col-lg-6 col-md-6  col-sm-6 col-6   mt-5  text-lg-start " >Project name 0
+                        <br><span class="text-lg-start role-title   ">Role Title</span>
+                     </div>             
+                    </div>
+                
+                 
+                 <p class="pt-5 project-page-text">I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to
+                add your own content and make changes to the font. I’m a great place for you to tell a story and let your users know a
+                little more about you.</p>
+               </div>
+               <div class="col-lg-5 col-md-5 col-12 col-sm-12  p-0  ">
+                  <img class="img-fluid  w-100 h-100"  src="https://static.wixstatic.com/media/11062b_6369a82539704f78b4fffed0ef058629~mv2_d_2346_1998_s_2.jpg/v1/crop/x_242,y_0,w_1590,h_1993/fill/w_322,h_404,al_c,q_80,usm_0.66_1.00_0.01/Leaflet.webp">
+               </div>
+           </div>
+         
+        </div>
+
+
+<div class="projects-box mx-lg-auto mx-sm-auto mx-auto ">
+    <div class="row mx-lg-auto my-4">
+        <div class="col-lg-7 col-md-7  col-12 bg-white  card-right-color">
+            <div class="row d-flex flex-lg-row align-items-center  text-start">
+                <div class="col-lg-1  p-0 pt-5 mt-5 blue-box  ">
```

Comment:
Watch out with all these Bootstrap utility classes (`mx-auto`, `p-0` etc). You're overriding the grid classes and also losing track of what is happening when. Some of these don't work, others are doing the same thing twice... Remember to use the 'inspect' function on your browser to see what CSS is _actually_ being applied.